<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataAcquisitionModule</class>
 <widget class="QWidget" name="DataAcquisitionModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据采集</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="capturedImagesGroup">
       <property name="title">
        <string>已采集图像</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QListWidget" name="dataListWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::EditTrigger::DoubleClicked|QAbstractItemView::EditTrigger::EditKeyPressed</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="iconSize">
           <size>
            <width>128</width>
            <height>96</height>
           </size>
          </property>
          <property name="resizeMode">
           <enum>QListView::ResizeMode::Adjust</enum>
          </property>
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="uniformItemSizes">
           <bool>true</bool>
          </property>
          <property name="selectionRectVisible">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="deleteImageButton">
            <property name="text">
             <string>删除选中</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearAllButton">
            <property name="text">
             <string>清空列表</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveImagesButton">
            <property name="text">
             <string>保存所有</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="loadImagesButton">
            <property name="text">
             <string>加载图像</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="previewGroup">
       <property name="title">
        <string>图像预览</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="previewLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>200</height>
           </size>
          </property>
          <property name="text">
           <string>请连接设备并开始预览</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
